docs/get-started.md

```diff
@@ -26,7 +26,7 @@ Use the npm script (the `docker-compose` command must be available to npm)
 npm run docker-run
 ```
 
-You will then see in this tabs the server's log
+You will then see in these tabs the server's log
```

Comment:
I got this as an automatic fix from my code editor. And I also checked if this change was correct. Because tabs is plural this needs to be these instead of this, so that is why I changed it.

Reply:
I read too quickly indeed, my bad